_requests/get/wateredtoday.ts

```diff
@@ -0,0 +1,28 @@
+import { VercelRequest, VercelResponse } from "@vercel/node";
+import { supabase } from "../../_utils/supabase";
+
+export default async function handler(
+	request: VercelRequest,
+	response: VercelResponse
+) {
+	const todayAtMidnight = new Date();
+	todayAtMidnight.setHours(0, 0, 0, 0);
```

Comment:
I tried it, gives some weird typescript errors. In general would be good to use a date library, but since we will maybe refactor this project, i will postpone this to some later point.